OPAL/tac/src/main/scala/org/opalj/tac/cg/android/AndroidEntryPointsFinder.scala

```diff
@@ -0,0 +1,54 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj
+package tac
+package cg
+package android
+
+import scala.collection.mutable.ArrayBuffer
+import scala.jdk.CollectionConverters.CollectionHasAsScala
+
+import org.opalj.br.Method
+import org.opalj.br.MethodDescriptor
+import org.opalj.br.ReferenceType
+import org.opalj.br.analyses.SomeProject
+import org.opalj.br.analyses.cg.EntryPointFinder
+import org.opalj.br.android.AndroidManifest
+import org.opalj.br.android.AndroidManifestKey
+import org.opalj.tac.fpcf.analyses.MethodDescription
+
+/**
+ * The AndroidEntryPointFinder considers specific methods of launcher Activity Clases as entry points.
+ * An activity is a launcher activity if it contains an intent filter with action "android.intent.action.MAIN"
+ * and category "android.intent.category.LAUNCHER". Requires Android Manifest to be loaded.
+ *
+ * @author Tom Nikisch
```

Comment:
As with the other class, I think if there are several authors, there should be several `@author` lines. In case this is mostly rewritten, you can also designate yourself as the only author.